client/my-sites/pages/page-card-info/index.jsx

```diff
@@ -85,15 +91,27 @@ function PageCardInfo( {
 						<span className="page-card-info__item-text">{ translate( 'Your latest posts' ) }</span>
 					</span>
 				) }
+				{ ! isFront && theme && (
```

Comment:
Could recommend a single descriptive variable here. Like `isThemeHomepage`

Reply:
Would `isFrontPage` work? It would similar to Core's [`is_front_page()`](https://developer.wordpress.org/reference/functions/is_front_page/).

Reply:
As long as it provides the intent you mean. From reading this it sounds like any page content associated with a theme demo homepage that is not currently set as the front page?

Reply:
Oh, you meant a single variable for `! isFront && theme`. Got it.

Reply:
Hm, I'm not sure combining the variables will serve us too well. `isFront` tells us if that page is currently set as the front page. `theme` is about if this page is based on a theme template and delivers the name. These are two separate conditions that need to apply for the label to show up